### DISCUSSION TITLE
Documentation of inserting multiple choice in Set-PnPListItem wrong

### DISCUSSION THREAD

**Discussion by Myjestic** [pnp/powershell#3601] [General]
Documentation on https://pnp.github.io/powershell/cmdlets/Set-PnPListItem.html says
Multi-Choice: -Values @{"MultiChoice" = "Choice 1","Choice 2"}
but it has to be
Multi-Choice: -Values @{"MultiChoice" = "Choice 1;#Choice 2"}

Or am I wrong? Only the second version works for me.

**Comment by AndersRask** [comment]
I tested it, and it works fine with an array as specified. Maybe you are using the old outdated module (SharePointPnPPowerShellOnline) and not PnP.PowerShell ?

`set-pnplistItem -List Documents -Identity 1 -Values @{MultiChoice="Choice #1","Choice #2"}`
